### DISCUSSION TITLE
Can`t use the "sample-script-with-chatbot.cs"

### DISCUSSION THREAD

**Discussion by Sagittaro** [ORelio/Minecraft-Console-Client#2238] [Q&A] [answered]
When I use the sample script from git, my MCC Console show these info:
```
[MCC] Starting compilation of: '715b625b-1020-4fd3-a830-52aeafb11fc1'
[MCC] Compilation done without any error.
[MCC] [ExampleBot] Sucessfully Initialized!
[MCC] Script failed to clean-up
```
No continue message show in the console. How to solve this problem?

**Comment by milutinke** [comment, marked as answer]
The bot has been loaded judging by the messages.
It is most likely that your server has a custom chat format, you will have to modify the regex for that specific chat format.
Read this: https://mccteam.github.io/guide/configuration.html#chat-format
